scripts/test/wallet/integration/default.nix

```diff
@@ -0,0 +1,30 @@
+{ localLib ? import ./../../../../lib.nix
+, config ? {}
+, numCoreNodes ? 4
+, stateDir ? localLib.maybeEnv "CARDANO_STATE_DIR" "./state-demo"
+, system ? builtins.currentSystem
+, pkgs ? import localLib.fetchNixPkgs { inherit system config; }
+, gitrev ? "123456" # Dummy git revision to prevent mass rebuilds
+, ghcRuntimeArgs ? "-N2 -qg -A1m -I0 -T"
+, additionalNodeArgs ? ""
+}:
+
+with localLib;
+
+let
+  demo-cluster = iohkPkgs.demoCluster.override {
+    inherit gitrev numCoreNodes stateDir;
+    keepAlive = false;
+  };
+  executables =  {
+    integration-test = "${iohkPkgs.cardano-sl-wallet-new}/bin/cardano-integration-test";
```

Comment:
How so?

Reply:
Yeah, I spent 4 hours debugging why I couldn't talk to the wallet API over https with `wallet-debug` enabled. Reverse engineering the test code, I realized it was http, not https. To update this, all that needs done is change http to https and remove `--wallet-debug` from the arguments to the wallet. It would also be good if you had parameters for cert/key instead of hard-coded to look in `scripts/tls-files`.

Reply:
Let's put this reminder in the code as a comment?

We do have an option for that; it's just commented out right now :o Can easily uncomment it if you'd like. It's in `wallet-new/integration/CLI.hs` around line 19.